src/css/mixins.scss

```diff
@@ -2,3 +2,102 @@
   height: 100%;
   width: 100%;
 }
+@mixin xs {
+   @media (min-width: #{$screen-xs}) {
+       @content;
+       h1 {
+         width: 100%;
+       }
+       .user-info {
+         width: 100%;
+       }
+       .search-bar {
+         width: 100%;
+       }
+       .all-cards {
+         display: grid;
+         grid-gap: 30px;
+         grid-template-columns: repeat(1, 1fr);
+         padding: 20px 0 20px 20px;
+         width: 95.5vw;
+       }
+       .main-header {
+         flex-flow: row wrap;
+       }
+   }
+}
+@mixin sm {
+   @media (min-width: #{$screen-sm}) {
+       @content;
+       h1 {
+         width: 100%;
+       }
+       .user-info {
+         width: 100%;
+       }
+       .search-bar {
+         width: 100%;
+       }
+       .all-cards {
+         display: grid;
+         grid-gap: 30px;
+         grid-template-columns: repeat(2, 1fr);
+         padding: 20px 0 20px 20px;
+         width: 95.5vw;
+       }
```

Comment:
There are multiple instances of this code.  We can refactor and put these multiple instances in a mixin with parameter.  This can dry up the overall code in the scss.  Overall, it looks great!